### ISSUE TITLE
Unknown Props with React 15.2.1

### ISSUE THREAD

**Issue by timothyallan** [malte-wessel/react-textfit#6]
The React update doesn't allow unofficial props anymore on DOM elements, and is spitting out:

```
Warning: Unknown props `forceSingleModeWidth`, `perfectFit`, `throttle`, `autoResize`, `onReady` on <div> tag. 
Remove these props from the element. 
For details, see https://fb.me/react-unknown-prop
in div (created by Textfit)
```

**Comment by ambethia** [comment]
@timothyallan Did you find a solution for this?

**Comment by timothyallan** [comment]
@ambethia Nope, just sucking it up for now...

**Comment by dani-z** [comment]
I'm having the same issue.

**Comment by seanmadi** [comment]
Also having this issue.

**Comment by loris** [comment]
@malte-wessel Hi! would you mind have a look at the referenced PR? (https://github.com/malte-wessel/react-textfit/issues/7), thanks!

**Comment by yohio** [comment]
Same issue here too

**Comment by malte-wessel** [comment]
Fixed in 0.1.9
